### PR TITLE
Remove helpers and fix download columns

### PR DESCRIPTION
**Pull request by nikolaystanishev**

### CODE REVIEW

**Comment by nikolaystanishev** [comment]
[SP23: [gpf] inRoles columns should be configured and added to download columns](https://trello.com/c/qzjYeTWU/1034-sp23-gpf-inroles-columns-should-be-configured-and-added-to-download-columns)

File: dae/dae/studies/variants_db.py [hunk 1]
```diff
@@ -14,18 +14,31 @@
 from dae.backends.impala.impala_helpers import ImpalaHelpers
 from dae.backends.impala.impala_variants import ImpalaFamilyVariants
 
+from dae.gene.gene_info_config import GeneInfoConfigParser
+from dae.gene.weights import WeightsLoader
```
Comment (lchorbadjiev) [inline dae/dae/studies/variants_db.py]: consider renaming 'WeightsLoader' to 'WeightsFactory'

File: dae/dae/variants/family.py [hunk 2]
```diff
@@ -60,6 +60,36 @@ def has_attr(self, item):
     def get_attr(self, item):
         return str(self.atts.get(item))
```
Comment (lchorbadjiev) [inline dae/dae/variants/family.py]: consider moving this code to Study/StudyWrapper

File: wdae/wdae/helpers/dae_query.py [hunk 3]
```diff
@@ -4,10 +4,6 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def columns_to_labels(columns, column_labels):
-    return [column_labels[column] for column in columns]
-
-
 def join_line(l, sep='\t'):
```
Comment (lchorbadjiev) [inline wdae/wdae/helpers/dae_query.py]: consider moving this function to StudyWrapper

File: dae/dae/studies/variants_db.py (hunk 1 above)
Comment (nikolaystanishev) [reply dae/dae/studies/variants_db.py]: This is added in TODO in gpf instance trello card

File: wdae/wdae/helpers/dae_query.py (hunk 3 above)
Comment (nikolaystanishev) [reply wdae/wdae/helpers/dae_query.py]: It is moved to dae_utils in dae/utils and downloading of 
    * Genotype Browser
    * Common Report families data
    * Gene Weights
is moved in dae module.

File: dae/dae/variants/family.py (hunk 2 above)
Comment (nikolaystanishev) [reply dae/dae/variants/family.py]: This is moved to Study

File: dae/dae/studies/genotype_browser_config_parser.py [hunk 4]
```diff
@@ -93,7 +93,7 @@ def _parse_pheno_column(cls, pheno_columns):
```
[the remark this answers is not in the document]
Comment (nikolaystanishev) [reply dae/dae/studies/genotype_browser_config_parser.py]: now pheno slots have format optional field as last parameter

**Pull request merged**